libs/ui-lib/lib/ocm/components/clusterConfiguration/operators/CnvHostRequirements.tsx

```diff
@@ -48,12 +57,10 @@ const CnvHostRequirements = ({ clusterId }: { clusterId: Cluster['id'] }) => {
             ? ` and ${masterRequirements?.diskSizeGb} storage space`
             : ''}
         </ListItem>
-        <RenderIf condition={!isSingleNode}>
-          <ListItem>
-            OpenShift Data Foundation (recommended for full functionality) or another persistent
-            storage service
-          </ListItem>
-        </RenderIf>
+        <ListItem>
+          {odfText}
+          {lvmsText} or another persistent storage service
```

Comment:
All of this could be the result of one function.

```js
if (OPCVersionGreaterThan4_15) {
    return "ODF, LVMS or another...";
} else {
  if (isSNO) {
    return "LVMS or another...";
  } else {
    return "ODF or another...";
  }
}
```

I think that covers all of the possibilities, right?

Reply:
True. I've changed it